browser-test/src/session_timeout.test.ts

```diff
@@ -0,0 +1,167 @@
+import {test, expect} from './support/civiform_fixtures'
+import {
+  enableFeatureFlag,
+  loginAsAdmin,
+  loginAsTestUser,
+  validateScreenshot,
+  validateToastMessage,
+} from './support'
+import {waitForHtmxReady} from './support/wait'
+import {
+  setupSessionTimeout,
+  verifyTimeoutModalBehavior,
+  verifyModalDismissal,
+  verifyAutoLogout,
+  advanceToSessionInactivityWarning,
+  advanceToSessionLengthWarning,
+  advanceToSessionTimeout,
+} from './support/session_timeout'
+
+test.describe('Session timeout warnings for Admin', () => {
```

Comment:
(blocking, question) The only difference here compared to the other two application options appears to primarily be the addition of the screenshots. Can this reuse the same calls as the others.?